clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/ops/servergroup/DeployKubernetesAtomicOperation.groovy

```diff
@@ -92,7 +93,8 @@ class DeployKubernetesAtomicOperation implements AtomicOperation<DeploymentResul
     def clusterName = serverGroupNameResolver.combineAppStackDetail(description.application, description.stack, description.freeFormDetails)
 
     if (description.kind) {
-      return deployController(credentials, clusterName, namespace)
+      def controllerName = description.sequence ? serverGroupNameResolver.generateServerGroupName(description.application, description.stack, description.freeFormDetails, description.sequence, false) : clusterName
```

Comment:
Maybe replicaSet and deployment should be supported too?

Reply:
Let me finish statefulset and daemonset first.